services/visual-studio-marketplace/visual-studio-marketplace-azure-devops-installs.service.js

```diff
@@ -35,29 +42,14 @@ module.exports = class VisualStudioMarketplaceAzureDevOpsInstalls extends Visual
   static get examples() {
     return [
       {
-        title:
-          'Visual Studio Marketplace - Azure DevOps Extension (Total Installs)',
-        pattern: 'total/:extensionId',
-        namedParams: { extensionId: 'swellaby.mirror-git-repository' },
+        title: 'Visual Studio Marketplace Installs - Azure DevOps Extension',
+        namedParams: {
+          measure: 'total',
```

Comment:
I was initially a little skeptical about using dropdowns here, but I think it's growing on me.